field/TinyLang/Field/F17.hs

```diff
@@ -0,0 +1,64 @@
+module TinyLang.Field.F17
+    (F17(..)) where
+
+import           Prelude
+import           Test.QuickCheck
+import qualified Text.Megaparsec.Char.Lexer   as L
+import           TinyLang.Field.Core
+import           TinyLang.Field.ParsableField
+import           TinyLang.Field.ParserUtils
+
+{-| A quick implementation of the field of seventeen elements for
+  testing.  We just wrap integers in a newtype and perform operations
+  modulo 17, with inverses hard-coded in a table. |-}
+
+newtype F17 = F17 { _unF17 :: Int }
```

Comment:
This is nitpicking, but the convention we use is to start field selectors of `newtype`s with `un`, i.e. without underscores, which we usually use for `data`.

Reply:
Done in #880a71c5.